feature_engine/outliers/artbitrary.py

```diff
@@ -91,6 +91,36 @@ class ArbitraryOutlierCapper(BaseOutlier):
     transform:
         Cap the variables.
 
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> from feature_engine.outliers import ArbitraryOutlierCapper
+    >>> np.random.seed(42)
+    >>> X = pd.DataFrame(dict(x1 = np.random.exponential(size = 100),
+    >>>                     x2 = np.random.randn(100)))
+    >>> aoc = ArbitraryOutlierCapper(max_capping_dict={'x1': 3, 'x2': 1},
+    >>>                             min_capping_dict={'x2': 0.2})
+    >>> print("Max value prior Arbitrary Capper:")
```

Comment:
I'd remove all the intermediate prints and just show the resulting dataframe after capping at 2 and 8